packages/integration-karma/scripts/karma-plugins/env.js

```diff
@@ -29,19 +29,19 @@ function createEnvFile() {
         fs.mkdirSync(DIST_DIR);
     }
 
-    const content = [
-        `window.process = {`,
-        `    env: {`,
-        `        NODE_ENV: "development",`,
-        `        COMPAT: ${COMPAT},`,
-        `        DISABLE_SYNTHETIC: ${DISABLE_SYNTHETIC},`,
-        `        FORCED_NATIVE_SHADOW_MODE_FOR_TEST: ${FORCED_NATIVE_SHADOW_MODE_FOR_TEST},`,
-        `        NATIVE_SHADOW_MODE: ${DISABLE_SYNTHETIC || FORCED_NATIVE_SHADOW_MODE_FOR_TEST},`,
-        `        NATIVE_SHADOW_ROOT_DEFINED: typeof ShadowRoot !== "undefined"`,
-        `    }`,
-        `};`,
-    ];
-    fs.writeFileSync(ENV_FILENAME, content.join('\n'));
+    fs.writeFileSync(
+        ENV_FILENAME,
+        `
+        window.process = {
+            env: {
+                NODE_ENV: 'development',
+                COMPAT: ${COMPAT},
+                NATIVE_SHADOW: ${!SYNTHETIC_SHADOW_ENABLED || FORCE_NATIVE_SHADOW_MODE_FOR_TEST},
+                NATIVE_SHADOW_ROOT_DEFINED: typeof ShadowRoot !== 'undefined'
+            }
+        };
+    `
```

Comment:
Thanks for simplifying this. minor: missing an indentation here?

Reply:
Formatting is handled by prettier.